pkg/proto/configuration/auth/auth.proto

```diff
@@ -27,3 +31,20 @@ message InstanceNameAuthorizer {
   // The empty string may be used to indicate all instance names.
   repeated string allowed_instance_name_prefixes = 1;
 }
+
+// SpiffeAuthorizer  maps instance names to Spiffe-Subject matchers.
+//
+// An example configuration might look like
+//  {
+// 	spiffe:{
+// 		instanceNameSubjectMap:{
+// 			"foo":{
+// 				allowedSpiffeIds:{"example.com":"workload-1"}
+// 			}
+// 		}
+// 	}
+// }
+message SpiffeAuthorizer {
```

Comment:
SPIFFE is supposed to be an abbreviation, right? If so, could we call this SPIFFEAuthorizer? The same applies to any of the associated types in Go code.